manifests/XMind/XMind 2020/10.2.1.yaml

```
@@ -0,0 +1,20 @@
+---
+
+Id: XMind.XMind 2020
```

```suggestion
Id: XMind.XMind2020
```

or 

```suggestion
Id: XMind.XMind-2020
```

or 

```suggestion
Id: XMind.XMind_2020
```

winget validate throws error. When you change this part, don't forget to change file path.

```
┌[yigitemres]
└[~\gitrepos]> winget validate --manifest .\xmind.yaml
Manifest validation failed.
Manifest Error: Invalid field value. Field: Id Value: XMind.XMind 2020 Line: 3, Column: 5
```

---

Thanks for your suggestion, I am confusing about how to name it actually.
Long time ago, xmind company had a classic mind map called xmind8, which is the original "xmind"
Then they have an updated version, which was a big change, and they called it xmind zen. they use "xmind zen" for at least two years.
and now it becomes xmind 2020 on their website. However, on the start menu, it only shows "xmind".
I am thinking "Xmind 2020" is too long and it is different from the start menu. but "xmind" might make old users confused.
Any suggestion for the name? feel free to directly modify my commit.
I would prefer the third suggestion. Since the id must be the same as the folder, I might have to make another PR later

---

You don't have to do another pr. Just rename the file and push it with proper id.

> Any suggestion for the name? 

If we talk about `Name` part, I think that "XMind 2020" is correct name for the app. If we talk about `Id` part it has to be something differentiate it from `XMind8` to `XMind2020` (because XMind8 can be added by anyone). We have several options for this:

1) If we name XMind2020 as XMind
   1) We have to name XMind8 as XMind8

2) If we name XMind2020 as XMind2020 (or similar)
   1) We can name XMind8 as XMind
   2) We can name XMind8 as XMind8

I think that explicitly indicating the app name is good choice for every manifest. If I create this file, I'm gonna use `XMind2020` path.